package.json

```diff
@@ -6,8 +6,13 @@
   "scripts": {
     "test:open": "npm run lint && cypress open",
     "lint": "eslint ./cypress/e2e/**/*.ts",
+    "clean:reports": "rm -rf ./cypress/reports/*",
+    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports",
     "test": "cypress run",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "pre": "npm run clean:reports",
+    "post": "npm run combine:reports && npm run generate:reports"
```

Comment:
as this will act as hooks, they need to specify as prefix the type of hook (pre or post) and then as suffix the name of the script associated to the hook, in this case test. So correct names should be:

- `pretest`
- `posttest`